packages/imperative/src/operations/src/Operation.ts

```diff
@@ -28,6 +28,9 @@ export type IOperationUndoCompleted =
 export type IOperationResultReady<T> =
     (output: T, operationResults: IOperationResult<any>) => void;
 
+/**
+ * @deprecated
+ */
 export abstract class Operation<T> implements ITaskWithStatus {
```

Comment:
Do we have a planned replacement for the `Operation` class going forward, or do we plan on removing it entirely? I'm not sure who is all using this as it is an abstract class, but we might want to elaborate on the long-term intentions in the deprecation message.

Reply:
From what I saw, nobody ever used this at all.

Reply:
That's fair - since its still being exported, do you think we should err on the side of caution and add something short such as "No replacement planned, will be removed in v4"? I don't think that many devs would use this class either, but I think it would be helpful to provide context in case a private repo or company plug-in is leveraging this class without our knowledge.

Reply:
I was planning on having that in the changelog, unless you think it needs to be in the typedoc too.